virttest/embedded_qemu.py

```diff
@@ -0,0 +1,105 @@
+"""
+Classes and functions for embedded qemu driver.
+"""
+import re
+import logging
+
+import aexpect
+from avocado.core import exceptions
+from avocado.utils import path
+from avocado.utils import process
+
+from virttest import utils_split_daemons
+
+from . import utils_misc
+
+try:
+    path.find_command("virt-qemu-run")
+    EMBEDDEDQEMU = "virt-qemu-run"
+except path.CmdNotFoundError:
+    EMBEDDEDQEMU = None
+
+if not utils_split_daemons.is_modular_daemon():
```

Comment:
this will be called each time this module is loaded, suggest put them into EmbeddedQemuSession where it will be checked when really use the utils.

Reply:
Updated.